parl/remote/grpc_heartbeat/heartbeat_server.py

```diff
@@ -72,16 +72,22 @@ def timeout_timer(self):
 
 
 class HeartbeatServerThread(threading.Thread):
-    def __init__(self, heartbeat_exit_callback_func):
+    def __init__(self,
+                 heartbeat_exit_callback_func,
+                 exit_func_args=(),
+                 exit_func_kwargs={}):
         """Create a thread to run the heartbeat server.
 
             Args:
                 heartbeat_exit_callback_func(function): A callback function, which will be called after the 
-                                                        heartbeat exit. There should be no arguments in the 
-                                                        function.
+                                                        heartbeat exit.
+                exit_func_args(tuple): the argument tuple for the heartbeat_exit_callback_func invocation. Defaults to ().
```

Comment:
Please update the comments for these two lines, following the same suggestions at the `hearbeat_client.py`.

Reply:
fixed